pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -68,3 +68,52 @@ for (left, right) in typecombinations
         end
     end
 end
+
+macro gap(str)
+    return EvalString(string(str))
+end
+
+export @gap
+
+function LoadPackageAndExposeGlobals(package::String, mod::String; all_globals::Bool = false)
+    mod_sym = Symbol(mod)
+    Base.MainInclude.eval(:(
+        module $(mod_sym)
+            import GAP
+        end
+    ))
+    mod_mod = Base.MainInclude.eval(:(Main.$(mod_sym)))
+    Base.invokelatest(LoadPackageAndExposeGlobals,package,mod_mod; all_globals=all_globals)
```

Comment:
Na, this is because of the world age of `mod_mod`. I am not completely sure why this needs to be done, but apparently because `mod_mod` was created in `Main` during the call of this function, the world age of `mod_mod` is higher than the function calls world age.